packages/miniserve/build.sh

```diff
@@ -0,0 +1,39 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/svenstaro/miniserve
+TERMUX_PKG_DESCRIPTION="Tool to serve files via HTTP"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.14.0
+TERMUX_PKG_SRCURL=https://github.com/svenstaro/miniserve/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=68e21c35a4577251f656f3d1ccac2de23abd68432810b11556bcc8976bb19fc5
+TERMUX_PKG_DEPENDS=libbz2
+TERMUX_PKG_BUILD_IN_SRC=true
+termux_step_pre_configure() {
+	rm -f Makefile
+	termux_setup_rust
+}
+termux_step_make() {
+	cargo build --jobs $TERMUX_MAKE_PROCESSES --target $CARGO_TARGET_NAME \
```

Comment:
Why overriding default termux_step_make? It should be enough to just rm Makefile and leave compilation step to build-package.sh.

Reply:
Would the defualt `termux_step_make` expose `$CARGO_TARGET_NAME` for further use?

Reply:
```
termux_step_pre_configure() {
    rm -f Makefile
}
```

Do not define termux_step_make.

Compilation is for standard Rust packages  is handled by https://github.com/termux/termux-packages/blob/4c47903280a2be3077d5c9ec8b398d31fe9a5cb1/scripts/build/termux_step_make_install.sh.

Every other changes should be done in termux_step_post_make_install.

Reply:
I've merged new package few minutes ago. It is written in Rust too. See its build.sh: https://github.com/termux/termux-packages/commit/4c47903280a2be3077d5c9ec8b398d31fe9a5cb1

Reply:
Sorry for bothering but do you have any ideas on this error? I've only left `rm -rf Makefile` and commented out all other lines.
```
Finished release [optimized] target(s) in 3m 37s
Replacing /data/data/com.termux/files/usr/bin/miniserve
error: failed to move `/data/data/com.termux/files/usr/bin/cargo-installNvP5qG/miniserve` to `/data/data/com.termux/files/usr/bin/miniserve`

Caused by:
  Is a directory (os error 21)
```
I think cargo install for this package should move some folder instead of the binary?

Reply:
That's because you have messed up your environment by
```
termux_step_make_install() {
	install -Dm755 -t "$TERMUX_PREFIX"/bin/miniserve \
		target/${CARGO_TARGET_NAME}/release/miniserve
}
```

This will create $TERMUX_PREFIX/bin/miniserve directory and place actual binary at $TERMUX_PREFIX/bin/miniserve/miniserve